components/Snackbar/tests/unit/MDCSnackbarMessageViewTests.m

```diff
@@ -123,7 +123,42 @@ - (void)testAccessibilityHintSetFromSnackbarMessageProperty {
                         self.message.accessibilityHint);
 }
 
-- (void)testSnackbarSetAccessibiltyViewIsModalForActionSnacbars {
+- (void)testSnackbarAccessibiltyViewIsModalByDefaultWithActions {
+  // Given
+  self.manager.internalManager.isVoiceOverRunningOverride = YES;
+  MDCSnackbarMessageAction *action = [[MDCSnackbarMessageAction alloc] init];
+  action.title = @"Tap Me";
+  self.message.action = action;
+
+  // When
+  [self.manager showMessage:self.message];
+  XCTestExpectation *expectation = [self expectationWithDescription:@"completed"];
+  dispatch_async(dispatch_get_main_queue(), ^{
+    [expectation fulfill];
+  });
+  [self waitForExpectationsWithTimeout:3 handler:nil];
+
+  // Then
+  XCTAssertFalse(self.manager.internalManager.overlayView.accessibilityViewIsModal);
```

Comment:
Shouldn't this be `XCTAssertTrue` given the test name? "ViewIsModalByDefaultWithActions"?